CHANGELOG.next.asciidoc

```diff
@@ -39,6 +39,17 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Auditbeat*
 
+- auditd: Add error.message to events when processing fails. {pull}30009[30009]
+
+*Filebeat*
+
+- tcp/unix input: Stop accepting connections after socket is closed. {pull}29712[29712]
+- Fix using log_group_name_prefix in aws-cloudwatch input. {pull}29695[29695]
+- aws-s3: fix race condition in states used by s3-poller. {issue}30123[30123] {pull}30131[30131]
+- Fix broken Kafka input {issue}29746[29746] {pull}30277[30277]
```

Comment:
It looks like you've picked up some extra changelog entries here

Reply:
Yeah, I've been told it happens, and I already left a comment that it should not be merged yet.